bin/kamstrup_prober.py

```diff
@@ -93,74 +93,88 @@ def get_register(self, register):
         return received_data
 
 
-def json_default(obj):
-    if isinstance(obj, datetime):
-        return obj.isoformat()
+def find_registers_in_candidates():
+
+    args = parse_args()
+    if args.registerfile:
+        candidate_registers_values = []
+        with open(args.registerfile, "r") as register_file:
+            old_register_values = json.load(register_file)
+        for value in old_register_values.iterkeys():
+            candidate_registers_values.append(int(value))
     else:
-        return None
+        candidate_registers_values = range(0x01, 0xFFFF)
```

Comment:
`0x01, 0xFFFF` should be named variables on module level

Reply:
I added these as port_start_range/port_end_range